recordings-bulk-actions/nodejs/recordings-bulk-action.js

```diff
@@ -47,7 +49,7 @@ client.loginClientCredentialsGrant(clientId,clientSecret)
 
 function createRecordingBulkJob(){
     return recordingApi.postRecordingJobs({
-        action: 'DELETE', // set to "EXPORT" for export action
+        action: 'EXPORT', // set to "EXPORT" for export action
```

Comment:
Are there other values that can be set here.  Otherwise, the comment is redundant.